index.js

```diff
@@ -276,15 +276,18 @@ ExtractTextPlugin.prototype.apply = function(compiler) {
 									compilation.errors.push(err);
 									return callback();
 								}
-								meta = module[NS];
-								// Error out if content is not an array and is not null
-								if(!Array.isArray(meta.content) && meta.content != null) {
-									err = new Error(module.identifier() + " doesn't export content");
-									compilation.errors.push(err);
-									return callback();
+								// TODO: Investigate "TypeError: Cannot read property '...\node_modules\extract-text-webpack-plugin' of null" in case of css errors and remove workaround check for module.meta[NS] below
```

Comment:
hmm... 😛 we would still need a test for this I guess, if you don't have this regression anymore (not urgent for you), mabe better close the PR ?